src/common/constants.ts

```diff
@@ -2,8 +2,10 @@
 export const APP_NAME = '5 Calls';
 export const APP_URL = 'https://5calls.org';
 // export const APP_URL = 'http://localhost:8090';
+export const API_URL = 'https://api.5calls.org/v1';
 export const ISSUES_API_URL = `${APP_URL}/issues/?all=true&address=`;
-export const REPORT_API_URL = `${APP_URL}/report`;
+export const REGISTER_CALL_API_URL = `${APP_URL}/report`;
+export const REPORT_API_URL = `${API_URL}/counts`;
```

Comment:
Names here are a bit unwieldy, maybe we should go `REPORT_API_URL` for the `/report` endpoint and the `COUNTS_API_URL` for `/counts`

Reply:
Good catch! This has been fixed in commit dbe2ca3